src/actinia_core/resources/raster_export.py

```diff
@@ -221,8 +221,15 @@ def _execute(self):
         self.required_mapsets.append(self.mapset_name)
         self._create_temporary_grass_environment(source_mapset_name="PERMANENT")
 
+        # COG bug in GDAL, see https://github.com/OSGeo/gdal/issues/2946 will be fixed in GDAL 3.1.4
+        # use r.out.gdal -c to avoid the bug
+        format = "COG"
```

Comment:
See https://github.com/mundialis/actinia_core/pull/108#issuecomment-696174769

Reply:
a r.out.gdal test using `format=COG createopt="BIGTIFF=YES,COMPRESS=LZW"` with a resulting .tiff size of ~12 GB was successful und could also be imported in QGIS without problems.